tests/projects.spec.ts

```diff
@@ -46,8 +47,8 @@ test("project edit configuration", async ({ page, lxdVersion }) => {
   } else {
     await page.locator("label").filter({ hasText: "Network zones" }).click();
   }
-  await page.getByText("Allow custom restrictions on a project level").click();
 
+  await assertTextVisible(page, "Allow custom restrictions on a project level");
```

Comment:
This needs to be a click, otherwise we can't do the changes further down in the file.